### ISSUE TITLE
ValueError with numpy 1.15.3

### ISSUE THREAD

**Issue by neishm** [neishm/fstd2nc#18]
Running fstd2nc with the latest numpy is giving me the following cryptic error message:
```
ValueError: invalid literal for int() with base 10: 'N'
```

It works fine with numpy 1.15.2, so this appears to be a recent problem.

**Edit:** here's the full stack trace
```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "fstd2nc/__main__.py", line 110, in <module>
    _fstd2nc_cmdline_trapped()
  File "fstd2nc/__main__.py", line 105, in _fstd2nc_cmdline_trapped
    _fstd2nc_cmdline (*args, **kwargs)
  File "fstd2nc/__main__.py", line 62, in _fstd2nc_cmdline
    buf = buffer_type(infiles, **args)
  File "fstd2nc/__init__.py", line 62, in __init__
    super(Buffer,self).__init__(filename, *args,**kwargs)
  File "fstd2nc/mixins/netcdf.py", line 143, in __init__
    super(netCDF_IO,self).__init__(*args,**kwargs)
  File "fstd2nc/mixins/netcdf.py", line 93, in __init__
    super(netCDF_Atts,self).__init__(*args,**kwargs)
  File "fstd2nc/mixins/removestuff.py", line 49, in __init__
    super(RemoveStuff,self).__init__(*args,**kwargs)
  File "fstd2nc/mixins/filter.py", line 46, in __init__
    super(FilterRecords,self).__init__(*args,**kwargs)
  File "fstd2nc/mixins/xycoords.py", line 326, in __init__
    super(XYCoords,self).__init__(*args,**kwargs)
  File "fstd2nc/mixins/vcoords.py", line 92, in __init__
    super(VCoords,self).__init__(*args,**kwargs)
  File "fstd2nc/mixins/series.py", line 102, in __init__
    is_series = (fields['typvar'] == b'T ') & ((fields['grtyp'] == b'+') | (fields['grtyp'] == b'Y') | (fields['grtyp'] == b'T'))
  File "local/lib/python2.7/site-packages/numpy/ma/core.py", line 3016, in __array_finalize__
    self._fill_value = _check_fill_value(self._fill_value, self.dtype)
  File "local/lib/python2.7/site-packages/numpy/ma/core.py", line 476, in _check_fill_value
    fill_value = np.array(fill_value, copy=False, dtype=ndtype)
ValueError: invalid literal for int() with base 10: 'N'
```

**Comment by neishm** [comment]
I think this is a slightly different problem.  It's complaining about a literal "N" character, not a newline.  The call to int() isn't even in the fstd2nc code, it's buried deep in a numpy call.  Will run some tests tomorrow to figure out what exactly is happening in numpy.

**Comment by neishm** [comment]
Looks like this is a manifestation of https://github.com/numpy/numpy/issues/12248.  It should be fixed in the upcoming numpy 1.15.4.  Don't know if it's worth doing a workaround on this end, could just update the dependencies to require `numpy<=1.15.2` for the time being.
